src/styles/for.scss

```diff
@@ -0,0 +1,18 @@
+@each $i in $month-length {
+  .calendar__length--#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+    display: none;
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::after {
+    content: "#{$i}";
+  }
+}
+
+@each $x,
+  $y in $day-list {
+  .calendar__day--#{$x} {
+    margin-left: #{$y}px;
+  }
+}
```

Comment:
Don't need to create a separate file for cycles. Here you create classes for calendar, so it should be on calendar file